### PR TITLE
Som 61 implement streamline new background classes for all pages

### PR DESCRIPTION
**Pull request by JensLegarthRyge**

### CODE REVIEW

File: src/layouts/MainLayout.vue [hunk 1]
```diff
@@ -130,26 +130,26 @@ export default {
...
           leftSideLinks.value = await getLeftDrawerLinks(user.value)
         }
       }
       profileDropdown.value = await getDropdownLinks(user.value)
-    };
+    }
 
     // Add event listener when the component is mounted
     onMounted(() => {
       document.body.addEventListener("click", handleBodyClick);
```
Comment (Johannes-Forsting) [inline src/layouts/MainLayout.vue]: her

File: src/layouts/MainLayout.vue [hunk 2]
```diff
@@ -130,26 +130,26 @@ export default {
 
     // On page load, make both profile dropdown and left-drawer links as needed
     const onPageLoad = async () => {
-      leftDrawerOpen.value = false;
-      const userId = localStorage.getItem("userId")
-      if(userId){
+      leftDrawerOpen.value = false
+      const userId = localStorage.getItem("userId");
```
Comment (Johannes-Forsting) [inline src/layouts/MainLayout.vue]: her

File: src/layouts/MainLayout.vue [hunk 3]
```diff
@@ -80,11 +80,11 @@
 </template>
 
 <script>
-import { onMounted, ref } from "vue";
+import { onMounted, ref } from "vue"
 import{ getDropdownLinks, getLeftDrawerLinks } from "src/service/utility";
```
Comment (Johannes-Forsting) [inline src/layouts/MainLayout.vue]: her

File: src/pages/administerUsersPage.vue [hunk 4]
```diff
@@ -201,7 +203,7 @@
   <notification-banner ref="notificationBanner"></notification-banner>
 </template>
 
-<script>import { onMounted, ref } from "vue";
+<script>import { onMounted, ref } from "vue"
 import { readAllUsersByHouseId, updateUserById, createUser, deleteUserById, readUserById } from "src/api/user"
 import { getUser, routeFrontPage } from "src/service/authentication"
 import { userDataValid } from "src/service/utility";
```
Comment (Johannes-Forsting) [inline src/pages/administerUsersPage.vue:209]: her

File: src/pages/administerUsersPage.vue [hunk 5]
```diff
@@ -325,26 +325,26 @@ export default {
...
 
 
     const onPageLoad = async () => {
-      user.value = await getUser();
+      user.value = await getUser()
       if(user.value.isAdmin !== true){
-        await routeFrontPage();
+        await routeFrontPage()
       }
       users.value = await readAllUsersByHouseId(user.value.houseId);
```
Comment (Johannes-Forsting) [inline src/pages/administerUsersPage.vue:343]: her

**Review by Johannes-Forsting — CHANGES_REQUESTED** [review]

**Comment by JensLegarthRyge** [comment]
Det burde være fikset

**Pull request merged**